### PR TITLE
Accessories fix

### PR DESCRIPTION
**Pull request by m98-m89**
Resolves 
temp fix for Accessories until bohemia fix it in next update
#### Changes proposed in this pull request: 
- 

- [x] I have tested my changes and corrected any errors found

### CODE REVIEW

File: Altis_Life.Altis/core/shops/fn_weaponShopSelection.sqf [hunk 1]
```diff
@@ -33,69 +35,40 @@ if ((uiNamespace getVariable ["Weapon_Shop_Filter",0]) isEqualTo 1) then {
...
-                    if (_bool) then {
-                        ((findDisplay 38400) displayCtrl 38407) ctrlEnable true;
-                    } else {
-                        ((findDisplay 38400) displayCtrl 38407) ctrlEnable false;
-                    };
-                } else {
-                    ((findDisplay 38400) displayCtrl 38407) ctrlEnable false;
-                };
+            //Accessories Menu
+            _itemArray = _item call BIS_fnc_compatibleItems;;
```
Comment (MartinezDeveloper) [inline Altis_Life.Altis/core/shops/fn_weaponShopSelection.sqf]: You have two semicolons

Comment (ruddelless287) [reply Altis_Life.Altis/core/shops/fn_weaponShopSelection.sqf]: Is it working? Is it tested? In addition to this, there is one more problem. When the doctor revives the person, the silencer, binoculars, flashlight do not come back. The remaining items are coming back.

Comment (that-edward) [reply Altis_Life.Altis/core/shops/fn_weaponShopSelection.sqf]: > In addition to this, there is one more problem. When the doctor revives the person, the silencer, binoculars, flashlight do not come back. The remaining items are coming back.

What has this got to do with this pull request?

Comment (ruddelless287) [reply Altis_Life.Altis/core/shops/fn_weaponShopSelection.sqf]: it broke down in what I mentioned because of this new coming update. can you help

**Review by DomT602 — APPROVED** [review]

**Pull request merged**